src/Folder.jsx

```diff
@@ -1,28 +1,51 @@
+import { useState } from 'react';
+
+import { getLink, getLinkDetail } from './apis/api';
+import { useFetchFolder } from './hooks/useFetchFolder';
+
 import SearchBar from './components/SearchBar';
-import Card from './components/Card';
+import CardList from './components/CardList';
+import FolderEdit from './components/FolderEdit';
+import FolderList from './components/FolderList';
+import FolderAddBtn from './components/FolderAddBtn';
 import './Folder.css';
 
-const Folder = ({items}) => {
-  if(!items || !items.links) {
-    return (
-      <div>파트 1에서 만들었던 랜딩 페이지 적용</div>
-    )
+const TOTAL_LIST_NAME = '전체';
+
+const Folder = () => {
+  const [folderList, links, setLinks] = useFetchFolder();
+  const [currentfolderName, setCurrrentFolderName] = useState(TOTAL_LIST_NAME);
+
+  const handleTotalListClick = async () => {
+    const { data } = await getLink();
+
+    setLinks(data);
+    setCurrrentFolderName(TOTAL_LIST_NAME);
+  }
+
+  const handleListClick = async (folderName, folderId) => {
+    const { data } = await getLinkDetail(folderId);
+
+    setLinks(data);
+    setCurrrentFolderName(folderName);
   }
 
   return (
     <div className='container-folder'>
       <SearchBar className='search-bar_ly'/>
-      <ul className='card-frame_ly'> 
-        {items.links.map(({ createdAt, imageSource, title, description, url, id }) => (
-          <Card key={id} 
-            imgSrc={imageSource}
-            title={title}
-            description={description}
-            createdAt={createdAt}
-            url={url}
-          />
-        ))}
-      </ul>
+      <div className='container-folder-and-cards'>
+        <FolderList 
+          folderList={folderList}
+          totalHandler={handleTotalListClick}
+          listHandler={handleListClick}
```

Comment:
컴포넌트가 props로 이벤트 핸들러를 받을때는 on- 으로 표현하는게 일반적이긴 해요~
```suggestion
          onTotalListClick={handleTotalListClick}
          onListClick={handleListClick}
```